pkg/utils/utils.go

```diff
@@ -170,6 +173,27 @@ func GetTrimmedGroupName(group string) string {
 	return group
 }
 
+// GetRestorePvcSizePercetage - get the percentage of size that need to increased during the kdmp restore PVC creation.
+// In the case failure, retunring zero to assume default 10% value.
+func GetRestorePvcSizePercetage() int {
+	fn := "GetRestorePvcSizePercetage"
+	kdmpData, err := core.Instance().GetConfigMap(drivers.KdmpConfigmapName, drivers.KdmpConfigmapNamespace)
+	if err != nil {
+		logrus.Warnf("%v: error in reading configMap [%v/%v]",
```

Comment:
warning, info and error are generally consumed by customer also. I suggest we should avoid functionNames in Info, warn and Errorf. We should limit function names to be printed with Tracef and Debugf

Reply:
Coverted it to Debugf information as we are not returning the error.